### PR TITLE
adding persistent

### PR DESCRIPTION
**Pull request by osamples**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by osamples** [comment]
@conda-forge/help-python The recipes have passed all checks and are ready for your review. Thanks!

**Comment by osamples** [comment]
@dopplershift These are ready for review - thanks!

File: recipes/persistent/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/zopefoundation/persistent/
+  summary: Translucent persistent objects
+  license: ZPL-2.1
+  license_file: COPYRIGHT.txt
```
Comment (dopplershift) [inline recipes/persistent/meta.yaml:42]: Here too:
```suggestion
  license_file: LICENSE.txt
```

File: recipes/persistent/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - cffi
+    - cython
```
Comment (dopplershift) [inline recipes/persistent/meta.yaml:22]: I don't see cython being used anywhere in here?
```suggestion
```

**Review by dopplershift — CHANGES_REQUESTED** [review]
Just a few minor things.

File: recipes/btrees/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/zopefoundation/BTrees
+  summary: Scalable persistent object containers
+  doc_url: https://pythonhosted.org/BTrees/
+  license: ZPL-2.1
+  license_file: COPYRIGHT.txt
```
[the remark this answers is not in the document]
Comment (dopplershift) [reply recipes/btrees/meta.yaml:43]: Oh I see now that the license needs the copyright notice. Ok, how about this instead:
```suggestion
  license_file:
    - COPYRIGHT.txt
    - LICENSE.txt
```

**Review by dopplershift — APPROVED** [review]

**Pull request merged**